### PR TITLE
docs(samples): modified comments in the samples and minor refactor

### PR DESCRIPTION
**Pull request by Sita04**
Changes:

1. Modified Auth implicit sample to use GCS instead of GCE.
2. Modified comments
3. Minor variable value refactoring

### CODE REVIEW

File: samples/snippets/src/main/java/AuthenticateImplicitWithAdc.java [hunk 1]
```diff
@@ -37,24 +40,21 @@ public static void main(String[] args) throws IOException {
   // credentials to use.
   public static void authenticateImplicitWithAdc(String project) throws IOException {
 
-    String zone = "us-central1-a";
-    // This snippet demonstrates how to list instances.
+    // This snippet demonstrates how to list storage buckets.
```
Comment (piaxc) [inline samples/snippets/src/main/java/AuthenticateImplicitWithAdc.java:43]: Please remove-- while strictly true, it's not really what the snippet is about (adc).

File: samples/snippets/src/main/java/VerifyGoogleIdToken.java [hunk 2]
```diff
@@ -26,9 +26,8 @@ public static void main(String[] args) {
     // The Google ID token to verify.
     String idToken = "id-token";
 
-    // The service name for which the id token is requested. Service name refers to the
-    // logical identifier of an API service, such as "pubsub.googleapis.com".
-    String targetAudience = "pubsub.googleapis.com";
+    // The service name for which the id token is requested.
```
Comment (piaxc) [inline samples/snippets/src/main/java/VerifyGoogleIdToken.java:29]: This is interesting. If you are validating an ID token, you should be checking what is in the token against what you expect, right? So wouldn't this be something like 'The expected target, for example, "https://example.com"'?

**Review by piaxc — CHANGES_REQUESTED** [review]

File: samples/snippets/src/main/java/AuthenticateImplicitWithAdc.java (hunk 1 above)
Comment (Sita04) [reply samples/snippets/src/main/java/AuthenticateImplicitWithAdc.java:43]: done.

File: samples/snippets/src/main/java/VerifyGoogleIdToken.java (hunk 2 above)
Comment (Sita04) [reply samples/snippets/src/main/java/VerifyGoogleIdToken.java:29]: We are not doing a manual check here. Instead we let the API verify by providing the details that we used during creating the ID token.
Hence, I would prefer to leave it as such.

**Review by piaxc — APPROVED** [review]
Thanks Sita!

**Review by lesv — APPROVED** [review]

**Pull request merged**